web/views.py

```
@@ -314,6 +317,47 @@ def signup_view(request):
+
+        # Transform the nested structure into something more user-friendly
+        to_delete = deleted_objects_collector.nested()
+        protected = deleted_objects_collector.protected
+
+        # Format the collected objects in a user-friendly way
+        model_count = {
+            model._meta.verbose_name_plural: len(objs) for model, objs in deleted_objects_collector.model_objs.items()
+        }
+
+        # Format as a list of tuples (model name, count)
+        formatted_count = [(name, count) for name, count in model_count.items()]
```

Ruff (0.8.2)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)